README.md

```diff
@@ -18,6 +18,42 @@ ArmoniK.Core provides services for submitting computational tasks, keeping track
 
 More detailed information on the inner working of ArmoniK.Core is available [here](https://aneoconsulting.github.io/ArmoniK.Core/).
 
+## Features availability
+
+As this repository is open-source and we follow trunk based development, all features will be integrated into main, in multiple small steps.
```

Comment:
As this repository is open-source and we follow trunk based development, all features will be integrated into main, in multiple small steps
```suggestion
As this repository is open-source and we follow trunk based development, all features will be integrated into main, in multiple small steps.
```